tools/distrib/gen_experiments_and_format.sh

```diff
@@ -15,7 +15,7 @@
 
 set -e
 cd $(dirname $0)/../..
-tools/codegen/core/gen_experiments.py $@
+tools/codegen/core/gen_experiments.py --check $@
```

Comment:
This removes the last of the expiry checks, I think. We'll need to set up some nightly CI (or some such) to announce expired experiments.

Reply:
Yeah... there's a robot of some sort needed... it's just not this one :)

Reply:
> https://github.com/grpc/grpc/blob/master/tools/codegen/core/gen_experiments.py#L38
> 
> In this line, the if condition should be something like `if "--check" in sys.argv:` or `if sys.argv[1] == "--check":`

Why?